doc/tutorial.md

```diff
@@ -251,10 +251,10 @@ let version =
     |> Gen.tuple3
     |> Gen.map (fun (ma, mi, bu) -> Version (ma, mi, bu))
 
-Property.print <| property {
+Property.print (property {
     let! xs = Gen.list (Range.linear 0 100) version
     return xs |> List.rev = xs
-    }
+})
```

Comment:
I would do either
```fsharp
property {
} |> Property.print
```
or
```fsharp
property {
}
|> Property.print
```
but this is fine too.

There are other cases like this.  I won't comment on them until we decide what is preferred here.

Reply:
I think I would prefer

```fsharp
let name =
    property {
        // ..
    }
Property.print name
```

Often times F# code plays chicken with the off-side rule and I'd rather just be blunt and direct.

This is actually one situation where I think `<|` makes perfect sense to use. Passing a CE directly to a function as the last (or only) parameter, when no other piping is being used.

```fsharp
Property.print <| property {
    // ...
}
```

Reply:
I am fine with anything here